code/src/test/kotlin/builtinTests/TestAllBuiltinsExist.kt

```diff
@@ -18,8 +18,8 @@ object TestAllBuiltinsExist : Spek({
                 "-",
                 "/",
                 "*",
-                "==",
-                "!=",
+                "equals",
+                "notEquals",
                 "<",
                 ">",
                 "&&",
```

Comment:
ville vi ændre dem her til 'and' & 'or' istedet for symbolerne?

Reply:
ps, syntes vi skulle overveje at ændre 'negated' tilbage til 'not' 
tror bare det er simplerer